R/pk-parameter.R

```diff
@@ -45,6 +45,10 @@ PKParameter <- R6::R6Class("PKParameter",
     dimension = function(value) {
       private$readOnlyProperty("Dimension", value, rClr::clrGet(private$.dimension(), "Name"))
     },
+    #' @field unit Unit of the PK-Parameter (Read-Only)
+    unit = function(value) {

```

Comment:
this is the base unit of the pk parameter (e.g. the value in which is will  be saved)